docs/src/variables.md

```diff
@@ -580,6 +587,44 @@ julia> x = @variable(model, [i=1:2], base_name="x", lower_bound=i, integer=true)
     Creating two named JuMP variables with the same name results in an error at
     runtime. Use anonymous variables as an alternative.
 
+## Constrained variables
+
+To create variables constrained to belong to the [`SecondOrderCone`](@ref) either
+of the following two methods can be used:
+```jldoctest constrained_variables; setup=:(model=Model())
+julia> @variable(model, x[1:3] in SecondOrderCone())
+3-element Array{VariableRef,1}:
+ x[1]
+ x[2]
+ x[3]
+
+julia> @variable(model, y[1:3])
+3-element Array{VariableRef,1}:
+ y[1]
+ y[2]
+ y[3]
+
+julia> @constraint(model, y in SecondOrderCone())
+[y[1], y[2], y[3]] ∈ MathOptInterface.SecondOrderCone(3)
+```
+The variables `x` are called *constrained variables*.
```

Comment:
This is not consistent with the definition in MOI, "constrained variables are the variables created with add_constrained_variable or add_constrained_variables" (http://www.juliaopt.org/MathOptInterface.jl/dev/apireference/#Variables-1). I'd like to have a clear definition. Is it any variable that belongs to *at least* one `SingleVariable` or `VectorOfVariables` constraint?

Reply:
Maybe the wording isn't clear but what I mean here is that `x` is called *constrained variables* as opposed to `y` (which is created free and then constrained) which isn't called like this.
Was that the confusion ?

Reply:
Ah, I misread this as saying that `y` as also a constrained variable. Perhaps clarify that `x` is a constrained variable while `y` is not.